src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -82,6 +82,7 @@
 #include "ngraph/op/fused/elu.hpp"
 #include "ngraph/op/fused/gemm.hpp"
 #include "ngraph/op/fused/group_conv.hpp"
+#include "ngraph/op/fused/hard_sigmoid.hpp"
```

Comment:
The operation already implemented in PR #2842

Reply:
That branch merged into this PR since it is dependent on it. I've separated this since I've another PR which also uses this (hardsigmoid as fused op).

Reply:
This one should come first.